static/app/types/hooks.tsx

```diff
@@ -206,6 +207,19 @@ export type SettingsHooks = {
   'settings:organization-navigation-config': SettingsConfigHook;
 };
 
+/**
+ * Feature Specific Hooks
+ */
+export interface FeatureSpecificHooks extends SpendVisibilityHooks {}
+
+/**
+ * Hooks related to Spend Visibitlity
+ * (i.e. Per-Project Spike Protection + Spend Allocations)
+ */
+export type SpendVisibilityHooks = {
+  'spend-visibility:project-settings-field': GenericProjectComponentHook;
+};
+
```

Comment:
Shouldn't this go into the `SettingsHooks` section?

Reply:
For Spend Visibility there will likely be a bunch more hooks that all live in different places and I think it might be more organized to scope them by feature, rather than where they might be used in Sentry, especially if that might change.